rflx/model/message.py

```diff
@@ -1128,6 +1131,17 @@ def __compute_field_condition(self, final: Field) -> expr.Expr:
             location=final.identifier.location,
         )
 
+    @property
+    def serialize(self) -> Dict[str, Any]:
+        return {
+            "kind": "Message",
+            "data": {
+                "identifier": self.identifier.serialize,
+                "structure": [l.serialize for l in self.structure],
+                "types": {f.name: t.identifier.serialize for f, t in self.types.items()},
+            },
+        }
+
```

Comment:
Are aspects (i.e. checksums) deliberately ignored?

Reply:
Yes, for now. The receiving side does not handle them and I didn't want to generate output that is not used. I'll add this as needed.